package.json

```
@@ -87,27 +87,27 @@
             "type": "object",
             "title": "AutoHotkey Plus Plus",
             "properties": {
-                "ahk++.compilePath": {
+                "ahk++.file.compilePath": {
                     "type": "string",
                     "default": "C:/Program Files/AutoHotkey/Compiler/Ahk2Exe.exe",
                     "description": "Path to the AHK compiler."
                 },
-                "ahk++.enableIntellisense": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Enable IntelliSense (Preview). Changes take effect after reload."
-                },
-                "ahk++.executePath": {
+                "ahk++.file.executePath": {
                     "type": "string",
                     "default": "C:/Program Files/AutoHotkey/AutoHotkeyU64.exe",
                     "description": "Path to the AHK runner."
                 },
-                "ahk++.helpPath": {
+                "ahk++.file.helpPath": {
                     "type": "string",
                     "default": "C:/Program Files/AutoHotkey/AutoHotkey.chm",
                     "description": "Path to the AHK Help document."
                 },
-                "ahk++.showDebugButton": {
+                "ahk++.language.enableIntellisense": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Enable IntelliSense (Preview). Changes take effect after reload."
+                },
+                "ahk++.ui.showDebugButton": {
```

VS Code automatically renders `ui` as `Ui` which I think looks really gross. Let's just show `Menu` for now. See [Extension API - configuration](https://code.visualstudio.com/api/references/contribution-points#contributes.configuration). I thought there was an issue for this in VS Code, but I've lost it.

```suggestion
                "ahk++.menu.showDebugButton": {
```